src/api-reference/direct-connects/hotel-service-4/v4.schemas.markdown

```diff
@@ -103,6 +103,7 @@ Hotel property object returned by hotel search.
 `position`|[`Geolocation`](#schemageolocation)|-|Geolocation of property.|
 `address`|[`Address`](#schemaaddress)|-|**Required** Address of property.|
 `leadRate`|[`LeadRate`](#schemaleadrate)|-|**Required** The lowest nightly rate averaged over the stay. Not required if `availabilityStatus` is `CLOSED_OUT` or `UNKNOWN`.|
+`totalPrice`|[`TotalPrice`](#schematotalpricehotelproperty)|-|**Required** Details about the total pricing associated with stay.
```

Comment:
Anything filled by supplier - we cannot mark it as required.

Reply:
Required was removed